### PR TITLE
OPENEUROPA-756: Fix PHP 5.6 compatibility

### PR DESCRIPTION
**Pull request by voidtek**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -8,17 +8,16 @@
     "php-http/httplug": "^1.1",
     "php-http/discovery": "^1.4",
     "php-http/client-common": "^1.7",
-    "symfony/config": "~2.2|^3|^4.0",
```
Comment (drupol) [inline composer.json]: These lines are out of scope. You should leave them as it is.
Composer will choose the best version automatically.

Comment (ademarco) [reply composer.json]: Yes but we cannot use `^4.0` as Drupal 8 supports maximum `3.4`.

Comment (drupol) [reply composer.json]: True, but pcas has nothing to do with Drupal 8... Dashboard team needs this.

Comment (ademarco) [reply composer.json]: Ok, good point, we will solve this differently then, I'll open a follow up next week.

**Comment by voidtek** [comment]
I approve the changes of @drupol  and I pushed a little commit to resolve typo on composer.json.
